torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3415,10 +3415,11 @@ def generator():
             # Shape of weight and bias should be the same as num_channels
             weight = make_arg(input_shape[1])
             bias = make_arg(input_shape[1])
+            kwargs = {'weight': weight, 'bias': bias} if eps is None else {'weight': weight, 'bias': bias, 'eps': eps}
```

Comment:
There was a note about making sure weight and bias are independent, but it looks like they already are (we make one for each time through the for loop. Let me know if I'm missing something but for now I just refactored it so it's clearer